charts/osm/templates/jaeger-deployment.yaml

```
@@ -22,9 +22,20 @@ spec:
       {{- include "restricted.securityContext" . | nindent 6 }}
       {{- end }}
       serviceAccountName: jaeger
-      nodeSelector:
-        kubernetes.io/arch: amd64
-        kubernetes.io/os: linux
+      affinity: 
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                - key: kubernetes.io/os
+                  operator: In
+                  values:
+                  - linux
+                - key: kubernetes.io/arch
+                  operator: In
+                  values: 
+                  - amd64
+                  - arm64
```

Since it looks like the jaegertracing/all-in-one image supports more archs, we might as well add them here:
```suggestion
                  values: 
                  - amd64
                  - arm64
                  - ppc64le
                  - s390x
```
https://hub.docker.com/r/jaegertracing/all-in-one/tags

---

Do you just want to remove arch and leave linux?